app/views/errors/forbidden.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='main'>
+  <h1>Woops!</h1>
+  <p>You do not have access to this page.</p>
+  <p>  Are you <%= link_to 'signed in', '/auth/google_oauth2' %>?</p>
```

Comment:
Should we just conditionally show the link?

```
<% if !current_user %>
  <p>This is most likely being caused because you are not <a href="/auth/google_oauth2">signed in</a>.
<% end %>
```

If you want to keep it as it is now the question mark would probably remain outside of the link.